README.md

```diff
@@ -0,0 +1,62 @@
+# messaging-android
+
+## Data Model
```

Comment:
Could we possibly add a brief overview to the top of the README instead of starting out with the data model? That this is an Android messaging library, that it uses the Flutter framework, Lantern Android integrates it, that it re-uses code from Signal's protocol, etc?

Reply:
Done. Let me know if there's anything else you'd like to see in there.